flocker/node/agents/blockdevice.py

```diff
@@ -1585,6 +1670,11 @@ def calculate_changes(self, configuration, cluster_state):
             local_state.devices, local_state.paths, configured_manifestations,
         ))
         deletes = self._calculate_deletes(configured_manifestations)
+
+        # FLOC-1874 - The API doesn't support resizing.  Drop this calculation
```

Comment:
I won't delete it. The loopback tests should still pass. But I have explicitly skipped the resize tests for AWS (like we do for OpenStack).